manifests/istio-telemetry/kiali/templates/configmap.yaml

```diff
@@ -34,14 +34,10 @@ data:
         url_service_version: http://istio-pilot.{{ .Values.global.configNamespace }}:8080/version
       tracing:
         url: {{ .Values.kiali.dashboard.jaegerURL }}
-{{- if .Values.kiali.dashboard.inclusterjaegerURL }}
-        in_cluster_url: {{ .Values.kiali.dashboard.inclusterjaegerURL }}
-{{- end }}
```

Comment:
don't we still want these if..end statements? Its possible the user set those settings to "" explicitly (perhaps because they don't have them installed).

Reply:
I don't think it makes a difference, if the user doesn't set anything then we'll just have empty strings, which in the end is the same. See the other variables like `url`: same here.